src/ScheduleService.Business/Workspace/EditWorkspaceCommand.cs

```diff
@@ -18,24 +24,45 @@ public class EditWorkspaceCommand : IEditWorkspaceCommand
   private readonly IEditWorkspaceRequestValidator _validator;
   private readonly IHttpContextAccessor _httpContextAccessor;
   private readonly IResponseCreator _responseCreator;
-  private readonly IAccessValidator _accessValidator;
+  private readonly IPatchDbWorkspaceMapper _mapper;
 
   public EditWorkspaceCommand(
     IWorkspaceRepository repository,
-    IAccessValidator accessValidator,
     IHttpContextAccessor httpContextAccessor,
     IResponseCreator responseCreator,
+    IPatchDbWorkspaceMapper mapper,
     IEditWorkspaceRequestValidator validator)
   {
     _repository = repository;
-    _accessValidator = accessValidator;
     _httpContextAccessor = httpContextAccessor;
     _responseCreator = responseCreator;
+    _mapper = mapper;
     _validator = validator;
   }
 
-  public Task<OperationResultResponse<bool>> ExecuteAsync(Guid id, JsonPatchDocument<EditWorkspaceRequest> request)
+  public async Task<OperationResultResponse<bool>> ExecuteAsync(Guid id, JsonPatchDocument<EditWorkspaceRequest> request)
   {
-    throw new NotImplementedException();
+    Guid modifiedBy = _httpContextAccessor.HttpContext.GetUserId();
+    DbWorkspace dbWorkspace = await _repository.GetAsync(id);
+
+    if (!(modifiedBy == dbWorkspace.CreatedBy))
```

Comment:
You can use !=

Reply:
why we need this check? modifiedBy and createdBy should be same?

Reply:
add admin too

Reply:
done